### PR TITLE
{bio}[GCC/12.3.0] BEDTools v2.31.0, BamTools v2.5.2

### PR DESCRIPTION
**Pull request by maxim-masterov**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/BEDTools/BEDTools-2.31.0-GCC-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    'docs',
+    'data',
+    'genomes',
+    'scripts',
+    'test',
+]
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['bedtools', 'pairToBed', 'mergeBed', 'bedToBam', 'fastaFromBed']],
+    'dirs': ['bin', 'docs', 'data', 'genomes', 'scripts', 'test'],
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/b/BEDTools/BEDTools-2.31.0-GCC-12.3.0.eb:48]: Could we here re-use the array defined above?
```suggestion
    'dirs': files_to_copy,
```

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
zen2-ubuntu-2304 - Linux Ubuntu 23.04 (Lunar Lobster), x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.11.2
See https://gist.github.com/SebastianAchilles/30af69471f1b380722ba0e4cce4d1916 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen2

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/f899b61b53ec0b890e48a40ad21ca89f for a full test report.

**Review by casparvl — APPROVED** [review]

**Comment by casparvl** [comment]
Going in, thanks @maxim-masterov!

**Pull request merged**